### PR TITLE
FE-1370: Update README to reflect Tokenlist V2 changes

### PR DESCRIPTION
**Pull request by tremarkley**

### CODE REVIEW

**Comment by tremarkley** [comment]
Current dependencies on/for this PR:
* master
  * **PR #346** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/346" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/346?utm_source=stack-comment).

File: README.md [hunk 1]
```diff
@@ -18,27 +17,37 @@ The Superchain token list is used as the source of truth for the [Optimism bridg
...
+If your automated checks failed, you can see the reason for the failure by downloading `validation-artifacts.zip`, unzipping it and opening the `validation_results.txt` file. To locate the `validation-artifacts` follow these steps:
+
+1. Click on the `Details` link for `Validate PR` check in your PR
+2. Click `Summary` in the left panel
+3. Find the section on the page labeled `Artifacts` and click on `validation-artifacts`
+4. After download of `validation-artifacts.zip`, unzip it and open `validation_results.txt`
+
+If you make changes and need to run the validation check again, you will need to wait for a reviewer to approve the checks to run again. However, if you do not want to wait for a reviewer to approve the checks to run again to see if the failures have been resolved, you can run the validation checks locally by running:
+```
+tsx ./bin/cli.ts validate --datadir ./data --tokens <data folder name (e.g. ETH)>
```
Comment (roninjin10) [inline README.md:46]:
```suggestion
npx tsx ./bin/cli.ts validate --datadir ./data --tokens <data folder name (e.g. ETH)>
```

Comment (roninjin10) [reply README.md:46]: This is a safer command to suggest since it works even if they haven't installed tsx globally

**Review by roninjin10 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tremarkley) [reply README.md]: nice, good suggestion

**Pull request merged**